site/docs/workflows/license_check.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 5
+---
+
+# License Check
+
+At VGV, we keep track of the rights and restrictions external dependencies might impose on Dart or Flutter projects.
+
+:::info
+The License Check functionality is powered by [Very Good CLI's license checker](https://cli.vgv.dev/docs/commands/check_licenses), for a deeper understanding of some [inputs](#inputs) refer to its [documentation](https://cli.vgv.dev/docs/commands/check_licenses).
+:::
+
+## Steps
+
+The License Check workflow consists of the following steps:
+
+1. Setup Dart
+2. Set SSH Key (if provided)
+3. Install project dependencies
+4. Check licenses
+
+## Inputs
+
+### `working_directory`
+
+**Optional** The path to the root of the Mason brick.
```

Comment:
👀 I don't think we should explain it as a mason brick but more the user side of it

Reply:
This is definitely a typo, will fix.